### ISSUE TITLE
Legislator Badge

### ISSUE THREAD

**Issue by andrewtremblay** [IndivisibleSomerville/its-our-call#34]
`/src/components/LegislatorBadge` has a placeholder (🔵/🔴/⚪️emojis) to represent legislator party, this does not match current designs.  

<img width="198" alt="screen shot 2018-03-19 at 2 39 37 pm" src="https://user-images.githubusercontent.com/954150/37615352-738494b2-2b83-11e8-8912-e550c4dc2c4f.png">

# Requirements
- [x] Get/create white svgs for each state ~(see if design has assets for this)~
- [x] Change the emojis to a real circle and add state svgs ~as an optional property~. 
- [x] ~Support more than two colors (Green party is a good edge-case)~

**Comment by fleenshop** [comment]
Re this one: "Support more than two colors (Green party is a good edge-case)" -- I don't think we need this. We group Independents in with the party they tend to caucus with. This has been fine when we've asked about it in user testing. Any reason that wouldn't apply equally to the Green party?

**Comment by andrewtremblay** [comment]
As long as we can easily infer party association through the data somewhere(e.g. `green or democrat are always blue, republicans or tea party are always red`), I'm totally fine with keeping it two colors. 

I'd like to avoid hard-coding more complicated exceptions (e.g. `any Independent party in Massachusetts not named 'tea party' = blue`... etc.), as it makes code maintenance a real headache.

**Comment by fleenshop** [comment]
I think that sounds fine... If at some point, there's an Independent who tends to vote Republican, would we still be able to go in and set that person's color to red?

**Comment by andrewtremblay** [comment]
I'll let @NCPtarmigan handle that question. It sounds out of scope of just the frontend. 

At the very least I think we'd want an additional field for inferredParty, or displayColorType.

**Comment by andrewtremblay** [comment]
I found a svg module for us states: https://coryetzkorn.github.io/state-svg-defs/ 
@rlmayfield if you think these are okay to use for the badge we can unblock this issue.

**Comment by rlmayfield** [comment]
@andrewtremblay those look good to me!

**Comment by andrewtremblay** [comment]
![screen shot 2018-04-18 at 3 03 16 am](https://user-images.githubusercontent.com/954150/38916487-8e761202-42b5-11e8-864e-ebd48526e8f4.png)
